server/ERNI.PBA.Server.Business/Handlers/Users/CreateUserHandler.cs

```diff
@@ -36,15 +36,15 @@ public async Task<bool> Handle(CreateUserCommand request, CancellationToken canc
                 throw new OperationErrorException(StatusCodes.Status409Conflict);
             }
 
-            if (string.IsNullOrWhiteSpace(request.FirstName.Trim()) || string.IsNullOrWhiteSpace(request.LastName.Trim()))
+            if (string.IsNullOrWhiteSpace(request.FirstName) || string.IsNullOrWhiteSpace(request.LastName))
             {
                 throw new OperationErrorException(StatusCodes.Status400BadRequest);
             }
 
             var user = new User
             {
-                FirstName = request.FirstName,
-                LastName = request.LastName,
+                FirstName = request.FirstName.Trim(),
+                LastName = request.LastName.Trim(),
                 Username = request.Email,
```

Comment:
We don't need to trim email address (username), do we?

Reply:
Yes, because we can't have trailing or leading whitespaces in name